src/index.html

```diff
@@ -7,7 +7,14 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
-
+  <script>
+      var global = global || window;
```

Comment:
Sure we need this? Can't find a global.varname in the app, Buffer and process also.

Reply:
Found a more elegant solution added this fix in the polyfills.ts